src/Wellcome.Dds/Wellcome.Dds.Auth.Web/Roles.cs

```diff
@@ -46,6 +47,8 @@ public bool HasCompletedRestrictedAccessForm
             => sierraRoles.Contains($"{RestrictedArchiveFieldTag}:True");
 
         public DateTime Expires { get; set; }
+        
+        public string[] BarCodes { get; set; }
```

Comment:
Is this prop used anywhere?

Reply:
Well I _was_ going to use it here...

https://github.com/wellcomecollection/iiif-builder/pull/42/files#diff-02665137d1b5210d32d75943e0a39930R29

If the username didn't pass the authentication API, does the barcode?
But, as mentioned, the username started working. 

I agree that it's a bit up in the air like that. 
We should add the extra code to validate by barcode, as users still might enter a barcode. I think we should wait until Sierra 5.2 is applied though, as per this discussion:

https://digirati.slack.com/archives/CBT40CMKQ/p1597929377015700

I'll add a note in this PR. The code works, but it should be revisited once Sierra 5.2 is running.